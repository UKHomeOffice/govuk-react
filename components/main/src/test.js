import React from 'react';
import { mount } from 'enzyme';

import Main from '.';

describe(Main, () => {
  let wrapper;

  it('renders without crashing', () => {
    wrapper = mount(<Main />);
  });

  it('can render a child node', () => {
    wrapper = mount(<Main><div>Exmaple</div></Main>);
  });

  it('matches wrapper snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
