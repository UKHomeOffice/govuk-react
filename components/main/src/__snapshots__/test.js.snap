// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main matches wrapper snapshot 1`] = `
.emotion-1 {
  padding-top: 30px;
  text-align: center;
}

.emotion-0 {
  max-width: 960px;
  margin-left: 15px;
  margin-right: 15px;
  text-align: left;
}

@media only screen and (min-width:641px) {
  .emotion-0 {
    margin-left: 30px;
    margin-right: 30px;
  }
}

@media only screen and (min-width:1020px) {
  .emotion-0 {
    margin: 0 auto;
  }
}

<Main>
  <Styled(div)>
    <div
      className="emotion-1"
    >
      <Styled(div)>
        <div
          className="emotion-0"
        >
          <div>
            Exmaple
          </div>
        </div>
      </Styled(div)>
    </div>
  </Styled(div)>
</Main>
`;
